travis/before-deploy.sh

```diff
@@ -0,0 +1,5 @@
+if [[ "$TRAVIS_TAG" = "" ]]; then exit 0; fi
+
+PLATFORM=$([ "$TRAVIS_OS_NAME" == "linux" ] && echo "linux64" || echo "$TRAVIS_OS_NAME")
```

Comment:
Yeah, that's good. I'm just suggesting we break it down into two fields, and use those everywhere when we create tag names like `bin-commitHash-os-cpu`.